packages/@webex/plugin-cc/src/types.ts

```diff
@@ -148,3 +148,24 @@ export interface SubscribeRequest {
 }
 
 export type EventResult = IAgentProfile;
+
+export const CHANNEL_NAME = {
+  TELEPHONY: 'telephony',
+  CHAT: 'chat',
+  SOCIAL: 'social',
+  EMAIL: 'email',
+} as const;
```

Comment:
Why are we using `as const` ? What does it do?

For this we can create `enum`.

Reply:
@rarajes2  This is the new way of creating enum. Please refer to this document: https://confluence-eng-gpk2.cisco.com/conf/display/WTWC/How+to+use+enums+and+alternatives